verifier/src/qe_identity.rs

```diff
@@ -296,13 +297,14 @@ impl SignedQeIdentityVerifier {
     ///   QE identity request from
     ///   <https://api.trustedservices.intel.com/sgx/certification/v4/qe/identity?update=standard>
     ///   When this is `None` verification will always fail.
-    /// - `time` - The current system time
+    /// - `time` - The current system time. A None value for time can be used in cases where the
+    ///   calling code is unable to provide time. In such cases, time validation must be skipped.
     ///   This is expected to be generated by:
     ///     ```ignore
     ///     let time = DateTime::from_system_time(SystemTime::now()).unwrap();
     ///     ```
     ///   or equivalent
-    pub fn new(key: Option<VerifyingKey>, time: DateTime) -> Self {
+    pub fn new(key: Option<VerifyingKey>, time: Option<DateTime>) -> Self {
```

Comment:
similar here IFF impl INto is used
```suggestion
    pub fn new(key: Option<VerifyingKey>, time: impl Into<Option<DateTime>>) -> Self {
```